### PR TITLE
Updating project to match new atomiq template

### PR DESCRIPTION
**Pull request by bquenin**
appcelerator/amp-project#239

Tests needs to be done through drone, but this require administrative access to the upstream repository which I don't have.

### CODE REVIEW

File: docker-compose.test.yml [hunk 1]
```diff
@@ -1,41 +0,0 @@
-version: "2"
```
Comment (subfuzion) [inline docker-compose.test.yml:1]: The lib template still has a docker-compose.test.yml file for being able to run a test from a docker container.

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +0,0 @@
-FROM atomiq/node:onbuild
```
Comment (subfuzion) [inline Dockerfile:1]: The lib template still has a Dockerfile for being able to run a test from a docker container.

File: dist/Kafka.js [hunk 3]
```diff
@@ -12,33 +12,15 @@ var _events2 = _interopRequireDefault(_events);
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```
Comment (subfuzion) [inline dist/Kafka.js]: The entire `dist` directory should be removed.

**Comment by bquenin** [comment]
`atomiq test` doesn't work on lib template, this is why I think we should stick to test using drone instead. However, as I mentioned in my PR, in order to run drone tests, you need to enable the project in drone, and, in order to enable the project in drone, you need to have write access to the repository (https://github.com/appcelerator/amp-kafka) which I don't have.

**Comment by subfuzion** [comment]
The lib template does support running tests and I've disabled master branch protection on `amp-kafka`

**Comment by subfuzion** [comment]
@bertrand-quenin Looking good, and I see you restored `Dockerfile`, but can you please add `docker-compose.test.yml` back to the repo as well? Thanks.

**Comment by subfuzion** [comment]
@bertrand-quenin Merging since others are blocked, but we will want to address this later. Thanks.

**Pull request merged**